src/contracts/misc/GhoSteward.sol

```diff
@@ -0,0 +1,171 @@
+import {DataTypes} from '@aave/core-v3/contracts/protocol/libraries/types/DataTypes.sol';
+import {PercentageMath} from '@aave/core-v3/contracts/protocol/libraries/math/PercentageMath.sol';
+import {GhoInterestRateStrategy} from '../facilitators/aave/interestStrategy/GhoInterestRateStrategy.sol';
+import {IGhoToken} from '../gho/interfaces/IGhoToken.sol';
+import {IGhoSteward} from './interfaces/IGhoSteward.sol';
+
+/**
+ * @title GhoSteward
+ * @author Aave
+ * @notice Helper contract for managing risk parameters of the GHO reserve within the Aave Facilitator
+ * @dev This contract must be granted `PoolAdmin` in the Aave V3 Ethereum Pool and `BucketManager` in GHO Token
+ * @dev Only the Risk Council is able to action contract's functions.
+ * @dev Only the Aave DAO is able to extend the steward's lifespan.
+ */
+contract GhoSteward is IGhoSteward {
+  using PercentageMath for uint256;
+
+  /// @inheritdoc IGhoSteward
+  address public constant AAVE_SHORT_EXECUTOR = 0xEE56e2B3D491590B5b31738cC34d5232F378a8D5;
+
+  /// @inheritdoc IGhoSteward
+  uint256 public constant MINIMUM_DELAY = 5 days;
+
+  /// @inheritdoc IGhoSteward
+  uint256 public constant BORROW_RATE_CHANGE_MAX = 0.0050e4;
+
+  /// @inheritdoc IGhoSteward
+  uint40 public constant STEWARD_LIFESPAN = 60 days;
+
+  /// @inheritdoc IGhoSteward
+  address public immutable POOL_ADDRESSES_PROVIDER;
+
+  /// @inheritdoc IGhoSteward
+  address public immutable GHO_TOKEN;
+
+  /// @inheritdoc IGhoSteward
+  address public immutable RISK_COUNCIL;
+
+  Debounce internal _timelocks;
+  uint40 internal _stewardExpiration;
+
+  /**
+   * @dev Only Risk Council can call functions marked by this modifier.
+   */
+  modifier onlyRiskCouncil() {
+    require(RISK_COUNCIL == msg.sender, 'INVALID_CALLER');
+    _;
+  }
+
+  /**
+   * @dev Constructor
+   * @param addressesProvider The address of the PoolAddressesProvider of Aave V3 Ethereum Pool
+   * @param ghoToken The address of the GhoToken
+   * @param riskCouncil The address of the RiskCouncil
+   */
+  constructor(address addressesProvider, address ghoToken, address riskCouncil) {
+    require(addressesProvider != address(0), 'INVALID_ADDRESSES_PROVIDER');
+    require(ghoToken != address(0), 'INVALID_GHO_TOKEN');
+    require(riskCouncil != address(0), 'INVALID_RISK_COUNCIL');
+    POOL_ADDRESSES_PROVIDER = addressesProvider;
+    GHO_TOKEN = ghoToken;
+    RISK_COUNCIL = riskCouncil;
+    _stewardExpiration = uint40(block.timestamp + STEWARD_LIFESPAN);
+  }
+
+  /// @inheritdoc IGhoSteward
+  function updateBorrowRate(uint256 newBorrowRate) external onlyRiskCouncil {
+    require(block.timestamp < _stewardExpiration, 'STEWARD_EXPIRED');
+    require(
+      block.timestamp - _timelocks.borrowRateLastUpdated > MINIMUM_DELAY,
+      'DEBOUNCE_NOT_RESPECTED'
+    );
+
+    DataTypes.ReserveData memory ghoReserveData = IPool(
+      IPoolAddressesProvider(POOL_ADDRESSES_PROVIDER).getPool()
+    ).getReserveData(GHO_TOKEN);
+    require(
```

Comment:
How could the strategy of GHO be `address(0)`?

Reply:
Just a sanity check to ensure GHO is listed in the pool and its address is not wrong.